apps/src/templates/forms/Checkbox.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component, PropTypes } from 'react';
+import color from "../../util/color";
+
+const styles = {
+  option: {
+    fontSize: 14,
+    color: color.charcoal,
+    fontFamily: '"Gotham 4r", sans-serif',
+    padding: 5
+  },
+  optionBig: {
+    fontSize: 18,
+    fontFamily: '"Gotham 7r", sans-serif',
+    color: color.charcoal,
+    paddingBottom: 10,
+    paddingTop: 10
+  },
+  checkbox: {
+    width: 25,
+    height: 25,
+    marginTop: -2
+  }
+};
+
+class Checkbox extends Component {
+  state = {
+    isChecked: false,
+  }
+
+  toggleCheckboxChange = () => {
+    const { handleCheckboxChange, label } = this.props;
+
+    this.setState(({ isChecked }) => (
+      {
+        isChecked: !isChecked,
+      }
+    ));
```

Comment:
I'm not sure I understand what's all happening here. Why not just `this.setState({isChecked: !this.state.isChecked})`

Reply:
It's a destructuring of a function parameter: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Unpacking_fields_from_objects_passed_as_function_parameter